timesync/ticker_test.go

```diff
@@ -75,7 +76,7 @@ func TestTicker_StartNotifying(t *testing.T) {
 	r.True(tr.started)
 	_, e = tr.Notify()
 	r.NoError(e)
-	r.Equal(3, mc.countToLayer)
+	r.Equal(2, mc.countToLayer)
```

Comment:
note that is is an counting of call to converter.TimeToLayer(). since there is one less call from timeSinceLastTick(), hence 1 less count.